.github/workflows/audit_service_tags.yml

```diff
@@ -88,31 +89,17 @@ jobs:
             docker compose -f docker-compose.test.yml run -e CHANGED_FILES=${{ env.CHANGED_FILES }} web bash \
               -c "CI=true DISABLE_BOOTSNAP=true bundle exec rake service_tags:audit_controllers_ci"
 
-  assign-failure-label:
```

Comment:
Removed this job and added the job as an additional step to prevent loading actions/checkout@v4 again